db/app.js

```diff
@@ -0,0 +1,17 @@
+const express = require("express");
+const {getAllTopics} = require("./controller");
+const app = express();
+
+app.use(express.json());
```

Comment:
No need for this yet - think about what code you actually need for this ticket!!!! !